wiki/Replay/en.md

```diff
@@ -264,7 +264,7 @@ It can be exported, or downloaded and watched immediately by request.
 When watching a server-side replay (excluding exported replays), the player of the replay's profile will receive an increment of one(1) in the "Replays Watched by Others" counter.
 If a new top 50 global record was added, the previous holder for \#50 position's server-side replay will be removed.
 
-To watch a server replay, an osu!account must be signed in and be connected to Bancho.
+To watch a server replay, an osu! account must be signed in and be connected to Bancho.
```

Comment:
I don't think we're supposed to put a space on this.

Reply:
`osu! account` is right.

https://osu.ppy.sh/help/wiki/Article_Styling_Criteria#osu!